beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -283,6 +284,10 @@ func (s *Server) handleAttestationsElectra(
 	data json.RawMessage,
 ) (attFailures []*server.IndexedVerificationFailure, failedBroadcasts []string, err error) {
 	var sourceAttestations []*structs.SingleAttestation
+	currentEpoch := slots.ToEpoch(s.TimeFetcher.CurrentSlot())
+	if currentEpoch < params.BeaconConfig().ElectraForkEpoch {
+		return nil, nil, errors.Errorf("Electra attestations have not been enabled, current epoch %d enabled epoch %d", currentEpoch, params.BeaconConfig().ElectraForkEpoch)
```

Comment:
```suggestion
		return nil, nil, errors.Errorf("electra attestations have not been enabled, current epoch %d enabled epoch %d", currentEpoch, params.BeaconConfig().ElectraForkEpoch)
```

Error strings shouldn't start with capitalization